article/app/views/fragments/articleAsideSlot.scala.html

```diff
@@ -0,0 +1,18 @@
+@(
+    shouldShowAds: Boolean,
+    articleAsideOptionalSizes: Seq[String],
+    isSticky: Boolean = true
+
+)
+@import conf.switches.Switches.CommercialSwitch
+
+@if(CommercialSwitch.isSwitchedOn && shouldShowAds) {
```

Comment:
👍 yeah! @regiskuckaertz said something similar 😁 

It would make more sense: we seem to often check these in tandem. Maybe a target for a future PR?